edge.py

```diff
@@ -44,5 +46,7 @@
         run_model_actions(args)
     elif args.command == "experiments":
         run_experiments_actions(args)
+    elif args.command == "misc":
```

Comment:
I wonder if it should be `config` instead? Similar to `gcloud`

Reply:
It may as well be I guess, I've put it as misc, since this is practically a one-off command. No other gets/sets for the config is implemented.

Reply:
Let's change to `config` in that case, because we have the option to extend this functionality in the future.

Reply:
Changed